tests/integration/tier0/non-destructive/firewalld-inhibitor/main.fmf

```diff
@@ -0,0 +1,18 @@
+summary: |
+    Running firewalld inhibitor
+description: |
+    Verify that the inhibitor is raised, when the firewalld service is running before
+    and during the conversion.
```

Comment:
@kokesak, @danmyway, the test needs to be updated because the conversion is inhibited only when the CleanupModulesOnExit config option is enabled.

Reply:
Done!